package.json

```diff
@@ -26,8 +26,12 @@
     "node": ">=8.0.0"
   },
   "devDependencies": {
+    "chai": "^4.1.2",
```

Comment:
Can you remove the `^` from these so that these dependencies don't get auto updated?